spec/providers/coreIntegration/tcpsocket.integration.src.js

```diff
@@ -74,7 +91,8 @@ module.exports = function (provider, setup) {
         socket.close(function () {}),
         client.close(function () {}),
         receiver.close(function () {}),
-        done()]);
+        done()
```

Comment:
Done (just put the closing of the array and promise call back on the same line to be consistent w/other uses.